### ISSUE TITLE
psalm doesn't flag up duplicate param hints

### ISSUE THREAD

**Issue by SignpostMarv** [vimeo/psalm#1400]
* https://psalm.dev/r/dd494a658e
* using 601f303
* as per phpstan/phpstan#1931

# snippet
```php
<?php declare(strict_types = 1);

/**
* @param mixed $val
*
* @param scalar|array|object|null $val
*/
function foo($val) : void {
	echo var_export($val, true);
}
```
## actual output
> Psalm output (using commit 601f303): 
> 
> No issues!

## expected output
> Psalm output (using commit 601f303): 
> 
> ERROR: InvalidDocblock - 8:1 - Duplicate @param in docblock for foo

**Comment by orklah** [comment]
@muglug Can you provide some hint as to what should be done here? (and maybe #2245 as it's closely related)

Do you see this as a new issue or as a new InvalidDocblock?

**Comment by weirdan** [comment]
Looks like `InvalidDocblock` to me.

**Comment by muglug** [comment]
@orklah I think this should go in `CommentAnalyser`. That's the part of the processing where we've parsed the docblock into blocks, but before we've done type analysis.
